app/views/studies/_initialize_metadata_form.html.erb

```diff
@@ -102,7 +102,7 @@
 
     function setMetadataConventionMessage(useMetadataConvention) {
         var message;
-        const advancedUrl = "https://singlecell.zendesk.com/hc/en-us/articles/360060610232-Metadata-File-Overview"
+        const advancedUrl = "https://singlecell.zendesk.com/hc/en-us/articles/4406379107355-Metadata-powered-Advanced-Search"
```

Comment:
Not a blocking concern, but a thought: of the 5 updates, this is the only one where the old link seems like it might be more appropriate.  The rest are search-result specific, but this is in an upload form for metadata files.  The extra information about what a metadata file is seems like it is still useful.

Reply:
I believe that the way this function is put together the "helpUrl", which is a link to the Metadata Convention zendesk page, will be part of the message if the convention is required or the user has chosen not to use the convention, the only time this Advanced-search link is alone in the messaging is when the convention is not required and the user has chosen to use the convention (presumably meaning they have some understanding already of what the feature is about?) so i think it is okay to keep to the previous choice of article topic.